networkit/cpp/algebraic/CSRMatrix.cpp

```diff
@@ -136,8 +136,8 @@ void CSRMatrix::setValue(const index i, const index j, const double value) {
 
     if (colIdx != none && colIdx >= rowIdx[i] && columnIdx[colIdx] == j) { // the matrix already has an entry at (i,j) => replace it
         if (value == getZero()) { // remove the nonZero value
-            columnIdx.erase(columnIdx.begin() + colIdx);
-            nonZeros.erase(nonZeros.begin() + colIdx);
+            columnIdx.erase(std::next(columnIdx.begin(), static_cast<int64_t>(colIdx)));
+            nonZeros.erase(std::next(nonZeros.begin(), static_cast<int64_t>(colIdx)));
```

Comment:
Why is this change done?

Reply:
I also thought about that one in the review for a moment. In general it is more versatile when using iterator arithmetic, so that seems ok in terms coding-style.

Reply:
And the cast is needed because the second parameter is of type `std::iterator_traits<InputIt>::difference_type` which is signed.

Reply:
And why is `std::next` needed?

Reply:
Hm I think I misinterpreted a warning, the only fix needed was the cast, not using `std::next`. I relaced the usages of `std::next` with just the `+` operator.